charts/verapdf-rest/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
+# This supports shell glob matching, relative path matching, and
+# negation (prefixed with !). Only one pattern per line.
+.DS_Store
+# Common VCS dirs
+.git/
+.gitignore
+.bzr/
+.bzrignore
+.hg/
+.hgignore
+.svn/
+# Common backup files
+*.swp
+*.bak
+*.tmp
+*.orig
+*~
+# Various IDEs
+.project
+.idea/
+*.tmproj
+.vscode/
```

Comment:
Why do you add so many files in a sub-directory .gitignore which never should be here? That makes no sense, and should only contain actual required ignore patterns.

Reply:
This file was created automatically by ````helm create```` 
It is a [helmignore](https://helm.sh/docs/chart_template_guide/helm_ignore_file/) file that is used by the ````helm package```` command. So could be of use down the line if the chart is packaged and distributed.